packages/mc-scripts/src/commands/config-sync.js

```diff
@@ -126,12 +128,35 @@ const configSync = async () => {
     return;
   }
 
-  // TODO: show diff (followup task)
+  const customAppLink = getMcUrlLink(
+    mcApiUrl,
+    fetchedCustomApplication.organizationId,
+    fetchedCustomApplication.application.id
+  );
+
+  // Sanitize SVG to be consistent with the one in MC
+  localCustomAppData.icon = sanitizeSvg(localCustomAppData.icon);
```

Comment:
Do we really need to sanitize the icon?
Since we will be just displaying it in the console, I can't see the security problem.

When creating the custom application in the Settings API, we are already [sanitizing its value](https://github.com/commercetools/merchant-center-services/blob/main/packages/settings/lib/resolvers/mutations/organization-extension/create-custom-application.ts#L69) before saving the data.

Reply:
I understand that the icon SVG is sanitised in the settings API. However, I'm sanitising the local config icon so that the diffing helper can differentiate if the icon is a new one or the previously sanitized icon saved in the mc.